### PR TITLE
Document CLI List, Disable and Enable Plugin.

### PR DESCRIPTION
**Pull request by LucasZdv**

### CODE REVIEW

**Comment by LucasZdv** [comment]
CLI Commands https://github.com/octobercms/october/pull/4127

File: console-commands.md [hunk 1]
```diff
@@ -121,6 +124,27 @@ October includes a number of commands for managing plugins.
...
+### Disable Plugin
+
+`plugin:disable` - Disable an existing plugin.
+
+    php artisan plugin:disable AuthorName.PluginName
+
+<a name="plugin-enable-command"></a>
+### Enable Plugin
+
+`plugin:enable` - Enable an disabled plugin.
```
Comment (w20k) [inline console-commands.md]: @LucasZdv shouldn't it be `a disabled`?

**Comment by LucasZdv** [comment]
Yes, sorry! I have already corrected it. f6e45b7a56a591f74103b5ca7e9555ba3332e5d9

**Comment by LukeTowers** [comment]
Waiting for corrections to be made in related PR

**Pull request merged**